app/views/foods/_form.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= form_with model: [ current_user, current_user.foods.build ] do |form| %>
+  <%#= form_with(model: food) do |form| %>
+  <% if food.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(food.errors.count, "error") %> prohibited this food from being saved:</h2>
+
+      <ul>
+        <% food.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name, style: "display: block" %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div>
+    <%= form.label :measurement_unit, style: "display: block" %>
+    <%= form.number_field :measurement_unit , min: 0, max: 1000%>
```

Comment:
- [ ] The measurement unit for the food should be a string such as 'grams', 'kg' etc. However, because you have made the form input a `number_field` it is impossible to put the correct measurement unit for the food. Please make this change 👍. Please see attached image below to see what that field should look like 👇 
![image](https://user-images.githubusercontent.com/46839548/210825705-2d09ef58-e509-4d3f-8f5d-bb1c790ac0fe.png)